cmd/web.go

```diff
@@ -422,6 +422,20 @@ func runWeb(ctx *cli.Context) error {
 			Post(bindIgnErr(auth.CreateRepoForm{}), repo.ForkPost)
 	}, reqSignIn)
 
+	// Check for hashtag requests in UBN repositories.
+	// => repo name must contain `-ubn-` or end with `-ubn`.
+	m.Group("/:username/:reponame(.+-ubn-.+|.+-ubn$)", func() {
+		m.Get("/hashtags", repo.Hashtags)
+		m.Get("/hashtags/:hashtag", repo.HashtagDisambiguation)
+	}, ignSignIn, context.RepoAssignment(), context.RepoRef())
+
+	// Redirect hashtag requests for ineligible repositories
+	m.Group("/:username/:reponame(.+[^-][^u][^b][^n][^-].+|.+[^-][^u][^b][^n]$)", func() {
```

Comment:
Fixed. I found in the documentation that a regex match takes precedence over a glob match, so I was able to replace the regex with *.*.